Updated etcd example to mention tar availability in pod (#1961)
kanisterio/kanister: examples/etcd/etcd-in-cluster/k8s/README.md

This document is going to show you how you can backup the ETCD cluster that is running as part of your Kubernetes control plane. The
commands are run into a cluster that is setup using [Kubeadm](https://github.com/kubernetes/kubeadm) but it should work on any other single or multi node ETCD cluster.

> **Note**
> 
> This blueprint will only work if the etcd pod has `tar` binary available as `kubectl cp` command
> in the blueprint, requires `tar` binary to be present on source.
> 

Reviewer: ```suggestion
**Note**

This blueprint will only work if the ETCD pod has `tar` binary available because `kubectl cp` command
in the blueprint, requires `tar` binary to be present on the ETCD pod.
```
